yt_dlp/extractor/iwara.py

```diff
@@ -79,13 +142,26 @@ def _extract_formats(self, video_id, fileurl):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        video_data = self._download_json(f'https://api.iwara.tv/video/{video_id}', video_id, expected_status=lambda x: True)
+        username, password = self._get_login_info()
+        if username and password:
+            headers = {
+                'Authorization': f'Bearer {self._get_media_token()}',
+                'Content-Type': 'application/json'
+            }
+        else:
+            headers = {
+                'Content-Type': 'application/json'
+            }
```

Comment:
```suggestion
        headers = {
            'Authorization': f'Bearer {self._get_media_token()}',
            'Content-Type': 'application/json'
        } if username and password else {  
            'Content-Type': 'application/json'
        }
```

Reply:
is `Content-Type` actually needed here? it's not POST

Reply:
Nah not needed lol